composer.json

```diff
@@ -7,7 +7,10 @@
         "ymcatwincities/openy": "8.2.*",
         "composer/installers": "^1.2",
         "drupal-composer/drupal-scaffold": "2.4.0",
-        "cweagans/composer-patches": "~1.0"
+        "cweagans/composer-patches": "~1.0",
+        "drupal/console": "1.9.2",
```

Comment:
I guess we don't need `"drupal/console-core": "1.9.2"` and `"drupal/console-en": "1.9.2"` as they are dependencies of `drupal/console` itself

Reply:
@AndreyMaximov we need them because if you would try to install drupal/console:1.0.0 or any other older version it'll install drupal/console-core:1.9.2 
I had more success on different versions of OPenY with providing exact version numbers

Reply:
But I'll try one more time

Reply:
You were right, it does work for initial install @AndreyMaximov 
Corrected